spyder/plugins/completion/kite/client.py

```diff
@@ -63,21 +63,26 @@ def stop(self):
     def get_languages(self):
         verb, url = KITE_ENDPOINTS.LANGUAGES_ENDPOINT
         success, response = self.perform_http_request(verb, url)
+        if response is None:
+            response = []
```

Comment:
I think this should also be `False, None`.

@andfoy?

Reply:
Sorry, I just saw that we're only returning response here. So I think this is ok?

Reply:
This is fine